src/client/api/init.c

```diff
@@ -239,8 +248,17 @@ daos_fini(void)
 	int	rc;
 
 	D_MUTEX_LOCK(&module_lock);
-	if (!module_initialized)
+	if (module_initialized == 0) {
+		/** calling fini without init, report an error */
 		D_GOTO(unlock, rc = -DER_UNINIT);
+	} else if (module_initialized > 1) {
```

Comment:
(style)  suspect code indent for conditional statements (8, 17)

Reply:
I don't get what is the problem here ...

Reply:
the 2 lines in the else  statement have 2 tabs + 1 space indented instead of just 2 tabs:
		 module_initialized--;
		 D_GOTO(unlock, rc = 0);

Reply:
ah, thanks. I was focused on the else if